src/pages/themes/[page].astro

```diff
@@ -86,7 +86,7 @@ const allThemes: ThemeAndAuthor[] = await fetch(
 // take all matching themes and create a paginated list of results
 const paginatedResults = paginate({
 	data: allThemes.filter((theme) => !theme.Theme.featured),
-	pageSize: 12,
+	pageSize: 24,
```

Comment:
I think we agreed on 18 rather than 24? The issue with really long pages is it can be a bit awkward scrolling up and down while the filter UI is still only up top.

```suggestion
	pageSize: 18,
```